DQM/CTPPS/test/totemt2_dqm_test_cfg.py

```diff
@@ -19,32 +21,44 @@
 process.load('Configuration.StandardSequences.Services_cff')
 process.load('FWCore.MessageService.MessageLogger_cfi')
 process.load('Configuration.EventContent.EventContent_cff')
-#process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
 
 # load DQM framework
 process.load("DQM.Integration.config.environment_cfi")
 process.dqmEnv.subSystemFolder = "CTPPS"
 process.dqmEnv.eventInfoFolder = "EventInfo"
 process.dqmSaver.path = ""
 process.dqmSaver.tag = "CTPPS"
-
-process.source = cms.Source('PoolSource',
+# raw data source
+process.source = cms.Source("NewEventStreamFileReader",
     fileNames = cms.untracked.vstring(
-        '/store/data/Run2018D/ZeroBias/RAW/v1/000/324/747/00000/97A72F4B-786F-5A48-B97E-C596DD73BD77.root',
-    ),
+        'http://cmsrep.cern.ch/cmssw/download/data/RecoPPS/Local/V1/run364983_ls0001_streamA_StorageManager.dat',
+#        '/store/group/dpg_ctpps/comm_ctpps/TotemT2/RecoTest/run364983_ls0001_streamA_StorageManager.dat',
+    )
 )
 
-#from Configuration.AlCa.GlobalTag import GlobalTag
-#process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_hlt_relval', '')
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, '130X_dataRun3_HLT_v2', '')
 
-process.load('CalibPPS.ESProducers.totemT2DAQMapping_cff')
+#Raw-to-digi
 process.load('EventFilter.CTPPSRawToDigi.ctppsRawToDigi_cff')
+
+#process.load('CalibPPS.ESProducers.totemT2DAQMapping_cff')
+
 process.load('Geometry.ForwardCommonData.totemT22021V2XML_cfi')
 process.load('Geometry.ForwardGeometry.totemGeometryESModule_cfi')
 process.load('RecoPPS.Local.totemT2RecHits_cfi')
 process.load('DQM.CTPPS.totemT2DQMSource_cfi')
+process.totemDAQMappingESSourceXML_TotemT2.verbosity = cms.untracked.uint32(0)
+process.totemT2Digis.RawUnpacking.verbosity = cms.untracked.uint32(0)
+process.totemT2Digis.RawToDigi.verbosity = cms.untracked.uint32(0)
+process.totemT2Digis.RawToDigi.useOlderT2TestFile = cms.untracked.uint32(1)
+process.totemT2Digis.RawToDigi.printUnknownFrameSummary = cms.untracked.uint32(0)
+process.totemT2Digis.RawToDigi.printErrorSummary = cms.untracked.uint32(0)
+process.totemDAQMappingESSourceXML_TotemT2.multipleChannelsPerPayload = cms.untracked.bool(True)
```

Comment:
Hello, @perrotta . I found that I had forgotten to add the newest parameter "useOlderT2TestFile" to EventFilter/CTPPSRawToDigi/python/totemT2Digis_cfi.py , so that line was left as is. Do you think I should change that other package in this PR?